app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/implicits/OgcImplicits.scala

```diff
@@ -93,21 +93,24 @@ class OgcImplicits[R: RenderableStore](layers: R, xa: Transactor[IO])(
 
   private def getSources(
       projectId: UUID,
-      tracingContext: TracingContext[IO]): IO[List[(OgcSource, List[CRS])]] =
+      tracingContext: TracingContext[IO]
+  ): IO[List[(OgcSource, List[CRS])]] =
     for {
       projectLayers <- ProjectLayerDao
         .listProjectLayersWithImagery(projectId)
         .transact(xa)
-      sourcesAndCRS <- projectLayers parTraverse { s =>
+      sourcesAndCRS <- projectLayers traverse { s =>
```

Comment:
This change was necessary because with `fs2-cron` as a dependency, this file can no longer find implicit evidence for `Parallel[IO]`. That makes me extremely nervous but we can't do anything about it because:

- there's no later version of `fs2-cron` on `fs2` 1.x for us to upgrade to
- we can't upgrade to `fs2-cron` on `fs2` 2.x, because that requires a cats upgrade as well, which is transitively impossible for us because Spark

I'm spending a little bit more time trying to figure out if I can restore that `Parallel[IO]` evidence somehow but not too much

Reply:
I tried. I don't know what to do. I don't think this is tractable in the short-term without either PR-ing to bump the version of fs2 used in a year old version of `fs2-cron` or making it possible to get to cats 2.x in gt dependencies

Reply:
Nevermind! The `Parallel[IO]` evidence is back! The problem was that `cron4s` 0.6.0 depended on `cats` 2.x, nothing to do with `fs2-cron` and the `fs2` 1.0.x mismatches at all. Shoutout sbt explicit dependencies for making the cats 1.6.1 eviction clear :sunglasses: